cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/CreateJarTask.java

```diff
@@ -42,16 +42,26 @@
 public class CreateJarTask implements Task {
 
     private boolean dumpBir;
+    private boolean buildNative;
+    private boolean dumpLlvmIr;
+    private boolean noOptimizeLlvm;
 
```

Comment:
shall we change the variable names as,
dumpLlvmIr -> dumpLLLVMIr
noOptimizeLlvm -> noOptimizeLLVM

Reply:
Refactoring in a seperate PR